### PR TITLE
tweaked update-all-versions.sh for drlx-parser

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

File: script/release/update-version-all.sh [hunk 1]
```diff
@@ -119,6 +119,11 @@ for repository in `cat ${scriptDir}/../repository-list.txt` ; do
             mvn -B -s $settingsXmlFile clean install -DskipTests
             returnCode=$?
 
+        elif [ "$repository" = "drlx-parser" ];then
+            #update version in drlx-parser/pom.xml
+            cd drlx-parser
+            mvn versions:set -DnewVersion=$newVersion
```
Comment (psiroky) [inline script/release/update-version-all.sh]: Would it make sense to call the function `mvnVersionsSet()`? It already has all the common config needed.

Comment (mbiarnes) [reply script/release/update-version-all.sh]: YES. Done

**Comment by mbiarnes** [comment]
Only makes sense when https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/543/files was merged

**Comment by mbiarnes** [comment]
close this - raise a new one for preventing future merge conflicts

**Comment by tarilabs** [comment]
Ref. https://github.com/kiegroup/drlx-parser/pull/6#discussion_r154640074

**Pull request closed without merging**